org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/groovy/SmartApp.groovy

```diff
@@ -93,34 +117,27 @@ def installed() {
 def updated() {
 	log.debug "Updated with settings: ${settings}"
 	unsubscribe()
-	initialize()
+	registerSubscriptions()
 }
 
 def initialize() {
 	state.connectionId = ""
-    state.endpointURL = "<PLACEHOLDER: Your HTTP endpoint to receive udpate notification>"
+	state.endpointURL = "<PLACEHOLDER: Your HTTP endpoint to receive udpate notification>"
 	registerSubscriptions()
 }
 
 //Subscribe events for all devices
 def registerSubscriptions() {
-	registerChangeHandler(switches)
-	registerChangeHandler(motionSensors)
-    registerChangeHandler(presenceSensors)
-    registerChangeHandler(contactSensors)
-    registerChangeHandler(waterSensors)
-    registerChangeHandler(locks)
-    registerChangeHandler(garageDoors)
-    registerChangeHandler(cameras)
+	registerChangeHandler(inputs)
```

Comment:
it's from vairable getInputs(). The usage is recommended by Samsung seems to be specific to SmartApp where it automatically maps "get" methods to a variable name. I.e. if you have a method called getInputs(), you can simplify calls to that by calling "inputs.xxxx"

---
In reply to: [100167546](https://github.com/openT2T/translators/pull/152#discussion_r100167546) [](ancestors = 100167546)